crates/gpui/src/platform/linux/platform.rs

```diff
@@ -213,15 +214,69 @@ impl Platform for LinuxPlatform {
         &self,
         options: PathPromptOptions,
     ) -> oneshot::Receiver<Option<Vec<PathBuf>>> {
-        unimplemented!()
+        let (done_tx, done_rx) = oneshot::channel();
+        self.foreground_executor()
+            .spawn(async move {
+
+                let dialog = AsyncFileDialog::new()
+                    .set_directory("./");
+
+                // XXX: Can options.files and options.directories both be true? RFD doesn't seem to
+                // support that.
+                if options.directories && options.files {
+                    unimplemented!();
+                }
```

Comment:
Open question here. Support for this might require patching RFD or something else.

Reply:
This is ok for now :)